openapi-public.yaml

```
@@ -1508,6 +1518,40 @@ components:
           description: "The ID of the newly created email address"
           allOf:
             - $ref: '#/components/schemas/UUID4'
+    JWTClaims:
+      type: object
+      description: "The claims extracted from a JWT."
+      properties:
+        subject:
+          description: "The unique identifier of the token's subject."
+          allOf:
+            - $ref: '#/components/schemas/UUID4'
+        issued_at:
+          description: "The timestamp indicating when the token was issued."
+          type: string
+          format: date-time
+        expiration:
+          description: "The timestamp indicating when the token will expire."
+          type: string
+          format: date-time
+        audience:
+          description: "The intended audience of the token."
+          type: string
+        issuer:
+          description: "The entity that issued the token."
+          type: string
+        email:
+          description: "The email address associated with the token's subject, if available."
+          type: string
+          format: email
```

This is currently an object with `address`, `is_primary`, `is_verified` so maybe something like (hope formatting works out):

```suggestion
          description: "Data about the email address associated with the token's subject, if available."
          type: object
          properties:
            address:
              description: "The actual email address."
              type: string
              format: email
            is_primary:
              description: "Indicates whether the email address is the primary address."
              type: boolean
            is_verified:
              description: "Indicates whether the email address is verified."
              type: boolean
```